src/Microsoft.ML/Models/OneVersusAll.cs

```diff
@@ -13,8 +13,20 @@ namespace Microsoft.ML.Models
     public sealed partial class OneVersusAll
     {
         /// <summary>
-        /// Create OneVersusAll multiclass trainer.
+        /// One versus all learner (also known as One vs the rest) is a multiclass learner with
+        /// strategy to fit one classifier per class. For each class it trains the provided 
+        /// binary classifier against all the other classes. 
+        /// See <a href="https://en.wikipedia.org/wiki/Multiclass_classification#One-vs.-rest">wikipedia</a> page.
         /// </summary>
+        ///<example>
+        /// In order to use it all you need to do is add it to pipeline as regular learner:
+        /// 
+        /// pipeline.Add(OneVersusAll.With(new StochasticDualCoordinateAscentBinaryClassifier()));
+        /// </example>
+        /// <remarks>
+        /// Underlying trainer suppose to be binary classfier. To check your options type BinaryClassifier
+        /// and look on all available learners suggested by IntelliSense.
+        /// </remarks>
         /// <param name="trainer">Underlying binary trainer</param>
         /// <param name="useProbabilities">"Use probabilities (vs. raw outputs) to identify top-score category</param>
```

Comment:
Are there any recommendations we can provide for when you'd want to choose raw vs. probabilities? Or perhaps a hint of the goals of why we have this parameter.

Reply:
Thank you Justin. Is this looks good for you?

---
In reply to: [199336245](https://github.com/dotnet/machinelearning/pull/447#discussion_r199336245) [](ancestors = 199336245)